db/migrate/20240603204215_add_application_to_reports.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddApplicationToReports < ActiveRecord::Migration[7.1]
+  def change
+    add_column :reports, :application_id, :bigint, null: true
```

Comment:
Column type is the same as on `statuses` — https://github.com/mastodon/mastodon/blob/b050eb6cdb7d120ea3db20db945556b44d15e56d/db/schema.rb#L1077